SYCL/ESIMD/api/functional/ctors/ctor_fill_fp_extra.cpp

```diff
@@ -0,0 +1,56 @@
+//==------- ctor_fill_fp_extra.cpp  - DPC++ ESIMD on-device test -----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// Test for simd fill constructor for extra fp types.
+// This test uses extra fp data types with different dimensionality, base and
+// step values and different simd constructor invocation contexts.
+// The test do the following actions:
+//  - construct simd with pre-defined base and step value
+//  - bitwise comparing expected and retrieved values
+
+#include "ctor_fill.hpp"
+
+using namespace esimd_test::api::functional;
+
+int main(int, char **) {
+  sycl::queue queue(esimd_test::ESIMDSelector{},
+                    esimd_test::createExceptionHandler());
+
+  bool passed = true;
+
+  const auto fp_types = get_tested_types<tested_types::fp_extra>();
+  const auto single_dim = values_pack<8>();
```

Comment:
Wouldn't it be better to use 'all_dimensions' here?

Reply:
For what reason we need to run testing on all dimensions?
@kbobrovs Konst, do you have any ideas maybe?

Reply:
Hm. I think what's there is enough. Slava is on vacation, so let's consider this resolved. He may have later comments, which can be addressed in a separate PR.

Reply:
@kbobrovs ok. thanks